edit/edit_test.go

```diff
@@ -775,7 +775,7 @@ func TestIndexOfRuleByName(t *testing.T) {
 	}{
 		{"first", 3},
 		{"%8", 4},
-		{"!package", 2},
```

Comment:
An integration test would be nice in https://github.com/bazelbuild/buildtools/blob/master/buildozer/buildozer_test.sh#L1170

Reply:
I didn't see that test previously. That explains why there wasn't a unit test for this. Should I go ahead and revert this unit test since it's not adding any real value above the integration test?

Reply:
Yes, sounds good.

Reply:
PTAL